compute/admin_guide/waas/deploy_waas/deployment_out_of_band.adoc

```diff
@@ -1,5 +1,7 @@
 == Deploy WAAS Out-Of-Band
 
+NOTE: A minimum 22.06 version of BOTH console and Defenders is required for Out-of-band.
```

Comment:
How about changing this as: 
"Out-Of-Band requires a minimum 22.06 version of both the Console and Defenders."

Reply:
Sounds good

Reply:
@divyabhushan - The sentence has been changed as per your suggestion

Reply:
@e-shuster , I moved it to the prereqs. Please take a look.

Reply:
> 
@e-shuster Thank you, and yes this line would be better placed in Prerequisites section at line number 14.

Reply:
@e-shuster This looks even better, thank you!